app/assets/json/translations/locale-en.json

```diff
@@ -442,6 +442,11 @@
         "APPLICANT_SEPARATE_ADDRESS": "Send my mail to a different address",
         "APPLICANT_SEXUAL_ORIENTATION": "How do you describe your sexual orientation or sexual identity?",
         "APPLICATION_CONTACT_DETAILS": "Application Contact Details",
+        "APPLICATION_IS_FIRST_TIME_HOMEBUYER": "I have not owned property within the past three years.",
+        "APPLICATION_HAS_COMPLETED_HOMEBUYER_EDUCATION": "I have completed homebuyers’ education.",
```

Comment:
minor edit on this. We just got feedback from MOHCD that it's actually *homebuyer education* rather than *homebuyers' education*